gobcore/model/metadata.py

```diff
@@ -14,6 +14,8 @@
 class FIELD:
     START_VALIDITY = "begin_geldigheid"
     END_VALIDITY = "eind_geldigheid"
+    START_VALIDITY_RELATION = "begin_geldigheid_relatie"
+    END_VALIDITY_RELATION = "eind_geldigheid_relatie"
     VERSION = "_version"
```

Comment:
I don't see the point why relations are any other than "normal" entities. Why not let relations have a start and end validity? Or should we also introduce VERSION_RELATION and other attributes? I suspect this change has some other reason?

Reply:
These fields are necessary in the GraphQL endpoints, as information about the relation is embedded in the related object. You're right about that this isn't actually a FIELD in the database, so this should be placed somewhere else, perhaps in the GraphQL code (only code using those names)

Reply:
I'll pick this up now.